tests/lib/plugin.js

```diff
@@ -82,7 +82,7 @@ describe("recommended config", () => {
                 // eslint-disable-next-line no-invalid-this
                 this.timeout(30000);
 
-                execSync("npm link && npm link eslint-plugin-markdown");
+                execSync("npm link && npm link eslint-plugin-markdown --legacy-peer-deps");
```

Comment:
We have to use `--legacy-peer-deps` because there is no version of `eslint-plugin-jsdoc` that supports ESLint 6, 7, and 8. `--legacy-peer-deps` disables additional peer dependency checks that were added in npm 8.6.0 and caused our CI to begin failing.